pyam/core.py

```diff
@@ -1382,6 +1355,66 @@ def map_regions(self, map_col, agg=None, copy_col=None, fname=None,
         if not inplace:
             return ret
 
+    def subtract(self, other, join_col, new_name, ignore_meta_conflict=False):
```

Comment:
Looking [pd.DataFrame.subtract()](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.subtract.html) for inspiration: how about renaming `join_col` to `axis`? If you think of timeseries data as multi-dimensional arrays, one could see `variable` or `region` as the axis...

Reply:
Nice!